frontend/src/components/Contexts/LanguageContext.tsx

```diff
@@ -46,12 +46,11 @@ export const LanguageProvider: FC<Props> = ({ children }) => {
 
 export const useLanguageContext = () => useContext(LanguageContext)
 
-export function Text(id: string) {
+export const translateText = (id: string): string => {
```

Comment:
We generally use capital letter for exported functions and lower letter for local ones